src/Components/Header/Header.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import "./Header.scss"
+import logoMobile from "../../assets/Logo/InStock-Logo_1x.png";
+
+const Header = () => {
+  return (
+    <header className='header'>
+      <img className='header-logo__img' src = {logoMobile}/>    
```

Comment:
i think the logo img tag  should also be in the nav tag right? Since is part of the navbar